client/components/editor/ActivityContent/index.vue

```diff
@@ -76,7 +76,8 @@ export default {
       return transform(elements, (acc, it) => {
         const comments = this.getComments({ activityId, contentElementId: it.id });
         const lastSeen = max([seen.contentElement[it.uid], seen.activity[activityUid]]);
-        acc[it.uid] = { ...it, comments, lastSeen: lastSeen || 0 };
+        const isResolved = !comments.length;
+        acc[it.uid] = { ...it, comments, isResolved, lastSeen: lastSeen || 0 };
```

Comment:
It doesn't feel like isResolved should be attached like this directly on content element without notion of comments.  If you gonna go down that path consider something like hasUnresolvedComments